MapleServer2/Packets/ClubPacket.cs

```diff
@@ -73,38 +101,61 @@ public static Packet UpdateClub(Player player, long clubId)
             pWriter.WriteInt();
             pWriter.WriteInt();
             pWriter.WriteLong(); // player house plot expiration timestamp
+            pWriter.WriteInt(); // combat trophy count
             pWriter.WriteInt(); // adventure trophy count
             pWriter.WriteInt(); // lifestyle trophy count
-            pWriter.WriteInt();
             return pWriter;
         }
 
         public static Packet ClubCreated(long clubId)
         {
             PacketWriter pWriter = PacketWriter.Of(SendOp.CLUB);
-            pWriter.WriteByte(0xF);
+            pWriter.WriteByte((byte)ClubPacketMode.ClubCreated);
             pWriter.WriteLong(clubId);
             pWriter.WriteInt();
             pWriter.WriteShort();
             return pWriter;
         }
 
-        public static Packet AssignLeader(Player player, long clubId, string clubName)
+        public static Packet Join(Player player, Club club)
         {
             PacketWriter pWriter = PacketWriter.Of(SendOp.CLUB);
-            pWriter.WriteByte(0x1E);
-            pWriter.WriteLong(clubId);
+            pWriter.WriteByte((byte)ClubPacketMode.Join);
+            pWriter.WriteLong(club.Id);
             pWriter.WriteUnicodeString(player.Name);
+            pWriter.WriteUnicodeString(club.Name);
+            return pWriter;
+        }
+
+        public static Packet Invite(long clubId, string clubName, Player player)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.CLUB);
+            pWriter.WriteByte((byte)ClubPacketMode.Invite);
+            pWriter.WriteLong(clubId);
             pWriter.WriteUnicodeString(clubName);
+            pWriter.WriteUnicodeString(""); //clubLeader
+            pWriter.WriteUnicodeString(player.Name);
+            return pWriter;
+        }
+
+        public static Packet InviteSentReceipt(long clubId, string invitee)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.CLUB);
+            pWriter.WriteByte((byte)ClubPacketMode.InviteSentReceipt);
+            pWriter.WriteLong(clubId);
+            pWriter.WriteUnicodeString(invitee);
             return pWriter;
         }
 
-        public static Packet EstablishClub(long clubId, string clubName)
+        public static Packet InviteResponse(long clubId, string clubName, string clubLeader, Player player, byte response)
         {
             PacketWriter pWriter = PacketWriter.Of(SendOp.CLUB);
-            pWriter.WriteByte(0x01);
+            pWriter.WriteByte((byte)ClubPacketMode.InviteResponse);
             pWriter.WriteLong(clubId);
             pWriter.WriteUnicodeString(clubName);
+            pWriter.WriteUnicodeString(""); //clubLeader
```

Comment:
Why not just put in the `clubLeader` variable here since it's already being passed in?